src/Riok.Mapperly/Descriptors/MappingBodyBuilders/BuilderContext/INewTupleExpressionBuilderContext.cs

```diff
@@ -0,0 +1,14 @@
+using Riok.Mapperly.Descriptors.Mappings;
+using Riok.Mapperly.Descriptors.Mappings.MemberMappings;
+
+namespace Riok.Mapperly.Descriptors.MappingBodyBuilders.BuilderContext;
+
+/// <summary>
+/// A <see cref="INewTupleExpressionBuilderContext{T}"/> for mappings which create the target object via ().
```

Comment:
I find the `... create the target object via ()` part confusing as the first time reading it I was not sure what `()` was referring to. Probably rephrase it to sth. like `... create the target tuple via a tuple expression (eg. (A: MapToA(src.A), B: MapToB(src.B)))`